content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md

```diff
@@ -194,6 +194,24 @@ Known issues:
 
 {{% capture discussion %}}
 
+## Garbage Collecting Load Balancers
+
+In usual case, the correlating load balancer resources in cloud provider should
+be cleaned up soon after a LoadBalancer type Service is deleted. But it is known
+that there are various corner cases where cloud resources are orphaned after the
+associated Service is deleted. Finalizer Protection for Service LoadBalancers was
+introduced to prevent this from happening. By using finalizers, a Service resource
+will never be deleted until the correlating load balancer resources are also deleted.
+
+Specifically, if a Service has Type=LoadBalancer, the service controller will attach
+a finalizer named `service.kubernetes.io/load-balancer-cleanup`.
+The finalizer will only be removed after the load balancer resource is cleaned up.
+Above ensures resource won't be leaked even in corner cases.
```

Comment:
I prefer the term "dangling" instead of "leaked in this scenario:
```
This prevents dangling load balancer resources even in corner cases such as the controller manager crashing. 
```

Reply:
Thanks, this is better.